README.md

```diff
@@ -1,82 +1,58 @@
 # Mainframer
 
-Tool that allows you to move build process from a local machine to a remote one.
+The tool that moves build process from a local machine to a remote one.
```

Comment:
"A tool that executes a command on remote machine while pushing and pulling files from/to local machine effectively implementing what is known as remote build/execution."?

Reply:
Rephrased it to put more emphasis on *execution* moving away from *build* since we are trying to be more abstract.